src/components/card/card.module.scss

```diff
@@ -8,20 +8,18 @@
 }
 
 .imageContainer {
-  position: relative;
-  padding-top: 56.6%;
```

Comment:
Why don't you use ratio 16:9 for image?

Reply:
I don't have any padding-top
![image](https://user-images.githubusercontent.com/107500715/220954616-78736478-fef9-4551-91ba-16be57dc559f.png)

Reply:
Done?